SokolTextGame/Commands.cs

```diff
@@ -27,6 +29,33 @@ public void Execute(string[] words, World world)
             Environment.Exit(0);
         }
     }
+    public class WhoAmI : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 1 && string.Join(" ", words) == "who") Console.Write("who, who? =)\n");
+            else if (words.Length == 3 && string.Join(" ", words) == "who am i") Console.Write( world?.player?.Description);
+            else Console.Write("I know the command \"who am i\"\n");
+        }
+    }
+    public class Look : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 1 && string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
+            else if (words.Length == 4 && string.Join(" ", words) == "look at my weapon") Console.Write(world?.player?.Weapon.Descriprion);
+            else Console.Write("I know the command \"look at my weapon\"\n");
+        }
+    }
+    public class WhereAmI : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 1 && string.Join(" ", words) == "where") Console.Write("Where? Where what? Or Who where?\n");
+            else if (words.Length == 3 && string.Join(" ", words) == "where am i") Console.Write(world.CurrentLocation().Description);
```

Comment:
Please remove all the Length checks from the conditions around here and only leave if(string.Join ...). They are actually redundant.